test/02.basic-querying.test.js

```diff
@@ -4,7 +4,7 @@ var db, User, Customer, AccessToken, Post, PostWithId, Category, SubCategory;
 
 /*eslint no-console: "off"*/
 /*global getSchema should*/
-describe('basic-querying', function () {
+xdescribe('basic-querying', function () {
```

Comment:
Why was this turned off and committed?
`+xdescribe('basic-querying', function () {`
cc @aquid @wolfgang-s